modules/configuring-otlp-output.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-output.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+The OpenTelemetry Protocol (OTLP) allows the cluster administrator to collect and forward logs to the OTLP receivers. The OTLP sends data over HTTP using JSON encoding by adhering to the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability Framework].
```

Comment:
I would modify the first sentence to only speak about the output.

"The OpenTelemetry Protocol (OTLP) output allows the cluster administrator to collect and forward logs to OTLP receivers. The OTLP output sends...."

Reply:
So the modified sentence may look like:

The OpenTelemetry Protocol (OTLP) output allows the cluster administrator to collect and forward logs to OTLP receivers. The OTLP output sends data over HTTP using JSON encoding by adhering to the [OpenTelemetry Observability Framework](https://opentelemetry.io/docs/specs/otlp/).